src/assets/js/promo.js

```diff
@@ -13,11 +13,15 @@ export default () => {
     endDate: '',
     now: dayjs(),
     hasSeenPromo: false,
+    promoID: '',
+
+    async init(startDate, endDate, promoID) {
+      this.promoID = promoID;
 
-    async init(startDate, endDate) {
       await this.setPromoDates(startDate, endDate);
 
-      localStorage.getItem('hasSeenPromo') === 'true'
+      localStorage.getItem('hasSeenPromo') === 'true' &&
```

Comment:
looks like we might not need this older mechanism anymore?

Reply:
Yes. Nice catch.